### DISCUSSION TITLE
3js and LivelyProperties

### DISCUSSION THREAD

**Discussion by SentientToast1** [rocksdanister/lively#2081] [Q&A]
anyone know how to make livelyproperties work with the latest version of 3js??

**Comment by rocksdanister** [comment]
The problem I was having is I am not sure how to execute the js function inside module using CefSharp.

As a workaround what I ended up doing is create global variables and used them in the threejs module:

https://github.com/rocksdanister/audio-visualizer-wallpaper/blob/3c76cf4c653fb06a55812d5347184f51502a6515/src/Living%20Room/index.html#L387

**Reply by SentientToast1** [reply]
So if i grab the js folder from the repo u send i could use 3js locally?

because my program itself is really simple and basic

**Reply by rocksdanister** [reply]
The included Room and Music Tv wallpapers are local threejs projects.

**Reply by SentientToast1** [reply]
Could u explain further. Sorry, im a bit new to modules in js

**Reply by rocksdanister** [reply]
Just download the project and play around with it, I just simply downloaded the threejs files online and loaded from disk.
